downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -38,8 +81,31 @@ To view other bug fixes, see the [GitHub milestone for Swan Lake 2201.6.0](https
 
 ### New features
 
+#### New Runtime Java APIs
+
+Introduced the `getInitMethod()` API in the `io.ballerina.runtime.api.types.ObjectType` class to get the method type of the initializer method of Ballerina objects.
```

Comment:
Shall we mention the method definition in java?

Reply:
It is not clear to me what you ask. Can you please suggest the change?

Reply:
sure
``` suggestion
Introduced the `getInitMethod()` API in the `io.ballerina.runtime.api.types.ObjectType` class to get the method type of the initializer method of Ballerina objects.
```java
    MethodType getInitMethod();
```

```

Reply:
Thanks! Fixed with 62048b5164f393ba3319993cca4c793084000709.